src/ExtranetBundle/LoginByTransferToken/LoginController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace ChameleonSystem\ExtranetBundle\LoginByTransferToken;
+
+use ChameleonSystem\CoreBundle\Service\PageServiceInterface;
+use ChameleonSystem\ExtranetBundle\Interfaces\ExtranetUserProviderInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+class LoginController
+{
+    /** @var TransferTokenServiceInterface */
+    protected $transferTokenService;
+
+    /** @var ExtranetUserProviderInterface */
+    protected $extranetUserProvider;
```

Comment:
They don't need to be protected.

Reply:
:heavy_check_mark: Using `private` in place of `protected`. Old habits coming through here.